pennylane/templates/utils.py

```diff
@@ -22,87 +22,79 @@
 from pennylane.variable import Variable
 
 
-def _check_no_variable(arg, arg_str, msg=None):
-    """Checks that ``arg`` does not contain a pennylane.Variable object.
+def _check_no_variable(arg, msg=None):
```

Comment:
Is there a case where passing no message to `_check_no_variable` is happening or admissible? If not I would just remove the `=None`.

Reply:
The same also applies to the other functions below

Reply:
Totally true, I overlooked that!